terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -123,7 +123,7 @@ resource "aws_network_acl_rule" "100_ssh" {
     rule_number = 100
     rule_action = "allow"
     from_port = 22
-    to_port = 22
+    to_port = 9300
```

Comment:
Note that in `118_ingestor_syslog` allows TCP traffic from `"${var.infra_cidr_all}"` while  SSH rule allows from 10.0.0.0/16 ( not sure why).